app/services/multi_tenant_service.rb

```diff
@@ -0,0 +1,17 @@
+class MultiTenantService
+  def initialize(service_org)
```

Comment:
```suggestion
  def initialize(service_org_name:)
```
This includes two suggestions, adding `_name` because it isn't clear what `service_org` is suppose to be. However, I think in this case we are thinking its going a string.

The other suggestion is to use keyword arguments. I prefer it because we stop having to rely on positional arugments which is hard to remember/know what they are. However, in this case we have only 1 parameter which isn't going to be a problem. The suggestion is up to you if you want to use keyword argument.